### ISSUE TITLE
site: mobile: nav: appears to have wrong css on first tap

### ISSUE THREAD

**Issue by frogtile** [json-ld/json-ld.org#769]
1. visit the site on mobile
2. tap on the top right icon for opening the nav
3. tap on "Specifications"

How does it look like?

now,

1. tap on the nav icon on to right again to close the nav
2. tap it on again to open it again
3. tap on "Specifications"

This time the items are visible

**Comment by davidlehn** [comment]
Yeah, looks like a bug.  The bootstrap css/js and layout html is from around 9+ years ago.  It could use a refresh, but I imagine it might take some effort to make sure the whole site keeps working.  If anyone wants to look at this particular bug and propose a fix, that would be great.
